core/scss/components/content-item/module.scss

```diff
@@ -1,13 +1,137 @@
 @import '../../tools/functions/core';
-
 @import '../../colors/settings';
 @import '../../settings/core';
 @import '../../settings/media';
 @import '../../typography/settings';
-@import '../../tools/functions/core';
+@import '../../tools/mixins/core';
+
+// // Settings
+@import '../../settings/settings';
 
+// // Tools
+@import '../../tools/mixins/baseline';
+@import '../../tools/mixins/border-radius';
 @import '../../tools/mixins/flex';
+@import '../../tools/mixins/focus';
 @import '../../tools/mixins/text-overflow';
-@import 'mixins';
+@import '../../typography/mixins';
+@include brand-font-face-regular;
+@include brand-font-face-light;
+@import '../../components/button/mixins';
+@import '../../components/loader/mixins';
+
+// // Component Helpers
+@include exports('md-button') {
+  [class|='#{$button__class}__container'] {
+    @include flex($display: inline-flex, $fd: column);
+
+    width: $button-container__width;
+
+    &[class$='--small'] {
+      width: $button-container--small__width;
+    }
+  }
+
+  // Default styles applied outside of media query
+  %#{$button__class},
+  .#{$button__class} {
+    @include button-base;
+    @include button-size(
+      $height: rem-calc(36),
+      $padding: $button__padding--36,
+      $radius: $button__radius--36
+    );
+    @include button-color($focus: true);
+    @include single-transition(background-color, $button__transition-speed, ease);
+
+    &--large,
+    &--52 {
+      @include button-size(
+        $height: rem-calc(52),
+        $padding: $button__padding--52,
+        $radius: $button__radius--52
+      );
+    }
+
+    &--circle {
+      @include radius($button__radius--round);
+
+      padding: 0;
+
+      @include button-circle-size($size: $button__size--36, $font-size: $button__font-size--36);
 
+      &[class*='--none'] {
+        @include button-color(
+          $color: inherit,
+          $bg: transparent,
+          $bg-hover: transparent,
+          $bg-active: transparent,
+          $color-disabled: $button__link-color--disabled
+        );
+        @include button-circle-size($size: auto);
+      }
+
+      &[class*='--32'] {
+        @include button-circle-size($size: $button__size--32, $font-size: $button__font-size--32);
+      }
+    }
+  }
+}
+
+@include exports('md-spinner') {
```

Comment:
We should have another discussion about the sub-components' CSS in the modules.

Reply:
Sounds good. There's definitely a compulsion to compare total parity when checking all the styles out, but I see the issue.